src/auth-service/utils/email.msgs.js

```diff
@@ -127,17 +127,35 @@ module.exports = {
   },
   user_updated: (firstName, lastName, updatedData) => {
     const updatedFields = Object.keys(updatedData)
-      .map((field) => `• ${field}: ${updatedData[field]}`)
+      .map((field) => `• ${field}`)
```

Comment:
Indicate what fields have been updated in the email BUT not their respective values